types/express-sanitized/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// Type definitions for express-sanitized 0.5
+// Project: https://github.com/askhogan/express-sanitized
+// Definitions by: Chris Barth <https://github.com/cjbarth/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as express from "express";
+
+declare function e(): (req: express.Request, res: express.Response, next: express.NextFunction) => void;
+
+declare namespace e {
```

Comment:
Don't do this. See https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this